docs/containers/debug-python.md

```diff
@@ -228,6 +228,126 @@ When you select **Docker: Add Docker Files to Workspace** for Django or Flask, w
 1. Modify and save any file.
 1. Refresh the browser and validate changes have been made.
 
+## How to build and run a container together
+
+1. In the previosly mentioned `tasks.json` file we have a dependency on the `docker-build` task. The task is part of the tasks array in the `tasks.json` and can look like this:
+
+```json
+"tasks":
+[
+  {
+    ...
+  },
+  {
+    "label": "docker-build",
+    "type": "docker-build",
+    "dockerBuild": {
+        "context": "${workspaceFolder}",
+        "dockerfile": "${workspaceFolder}/Dockerfile",
+        "tag": "YOUR_IMAGE_NAME:YOUR_IMAGE_TAG"
+    }
+  }
+]
+```
+
+> [!TIP]
```

Comment:
The VS Code website build doesn't handle this Markdown extension. You can just add a bolded prefix to the comment directly:
> **Tip**: As the dependency.....

Reply:
Updated it, thanks for the tip!